src/main/java/com/ericsson/ei/handlers/MongoDBHandler.java

```diff
@@ -394,12 +446,18 @@ public boolean checkDocumentExists(String databaseName, String collectionName, S
      * @return 
      */
     public boolean updateDocumentAddToSet(String dataBaseName, String collectionName, String condition, String eventId) {
-        try {
+    	LOGGER.info("=======EventObjectMap======= updating the document with eventid: "+ eventId);
```

Comment:
I guess this is logged often, change to logger debug.

Reply:
WIP